pkg/rules/api/v1_test.go

```diff
@@ -66,7 +66,7 @@ type rulesRetrieverMock struct {
 	testing *testing.T
 }
 
-func (m rulesRetrieverMock) RuleGroups() []thanosrule.Group {
+func (m rulesRetrieverMock) RuleGroups() []manager.Group {
```

Comment:
same here: let's call this alias `rulesmanager` so the context for `rulesmanager.Group` is not lost.